edward/inferences/inference.py

```diff
@@ -169,11 +171,25 @@ def initialize(self, n_iter=1000, n_print=None, scale=None, logdir=None,
     logdir : str, optional
       Directory where event file will be written. For details,
       see ``tf.summary.FileWriter``. Default is to write nothing.
+    logrun : str, optional
```

Comment:
Also: if logrun can easily be specified as part of logdir that would also enable the timestampping for those who prefer to control inference using `initialize()` then `updates()`. (that is, they don't call `run()`)